service/ztests/curl-load-line.yaml

```
@@ -0,0 +1,23 @@
+script: |
+  source service.sh
+  zed create -q test
+  curl -w 'code %{response_code}\n' -X POST -H 'Accept: application/json' \
+    -H 'Content-Type: application/x-line' \
```

Nit: I don't see any reason to prefer JSON over ZSON.
```suggestion
  curl -w 'code %{response_code}\n' -X POST -H 'Content-Type: application/x-line' \
```

---

But since you've merged this, don't worry about it!